PermanentOrg/web-app
# Fix Sass deprecation warnings

As a result of the upgrade to Angular 12, we now have a LOT of warnings for Sass for using the `/` operator which will become deprecated soon. This was brought up in #136, but we decided to ignore it. However, these deprecation warnings end up taking up a lot of space in logs when compiling and testing, so when tests fail (especially on Github Actions) it's hard to find where the actual error is. Fortunately, Sass provides an easy migration tool to fix all of these in preparation for future versions of Sass. This PR bumps up the version of Sass (so it's more explicitly set to a current version that is set up for the transition to the new format instead of just relying on transitive dependencies) and then runs that migration tool.

To test:
* Build the project and confirm there are no more deprecation warnings
* Make sure that none of the resulting CSS/styling is affected by these changes.

Resolves #139.
